src/components/global-view-core.js

```diff
@@ -238,6 +248,7 @@ const GlobalViewCore = ({ timestampMin, timestampMax, timestampStep }) => {
 
     return (
         <div>
+            {createWS()}
```

Comment:
Can you create the SockJsClient directly and define method that returns only the list of topics ?

Reply:
The createWS function has only two instructions, which could be written in two lines without the very very strict check-style.
I will split the websocket functionality across the file